File: Typescript/src/findDegrees.ts
export function findDegrees(matrix: number[][]): number[] {
	const degrees: number[] = Array(matrix.length);
	for (let i = 0; i < matrix.length; i++) {
		degrees[i] = matrix[i].reduce((sum, value) => sum + value, 0);
	}
	return degrees;
};

File: Cs/Solution/FindDegrees.cs


using System.Linq;

namespace Solution
{
	public partial class Solution
	{
		public int[] FindDegrees(int[][] matrix)
		{
			int n = matrix.Length;
			int[] degrees = new int[n];
			for (int i = 0; i < n; i++)
			{
				degrees[i] = matrix[i].Sum();
			}
			return degrees;
		}
	}
}
